### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by fpsapc**
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5).
- Deleting a task will update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) will be set to 1).
- All changes to the To-Do List will be saved in local storage.

### CODE REVIEW

File: src/modules/CrudMethods.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      deleteBtn.addEventListener('click', () => this.removeTask(task.id));
+      if (task.status) {
+        taskElement.querySelector('span').style.textDecoration = 'line-through';
+      }
+      this.taskList.appendChild(taskElement);
+    });
+  }
+}
+
+export default TaskList;
```
Comment (Whoistolu) [inline src/modules/CrudMethods.js:73]: - [ ] [REQUIRED] Hello  Khan, you quite have. an awesome UI design here 👍🏽 
However, it looks like you skipped the part of the project requirement that states that you need to add a function for editing task descriptions. Kindly implement this 🥇 

- [ ] [REQUIRED] Also, we need to ensure that the new tasks have the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5) and let's also take note that ...
- [ ] [REQUIRED] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

Let's fix these 💪🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @fpsapc 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/CrudMethods.js (hunk 1 above)
Comment (fpsapc) [reply src/modules/CrudMethods.js]: @Whoistolu I made all the required changes. Now I am going for a second review. Thanks.

Comment (Whoistolu) [reply src/modules/CrudMethods.js]: Awesome 👏 
@fpsapc

**Pull request merged**